resources/views/layouts/navbar.blade.php

```diff
@@ -56,4 +56,14 @@ function dismiss() {M.Toast.dismissAll();};
     </script>
     <li><a href="mailto:root@eotvos.elte.hu?Subject=[urán%20bug]" onclick="M.toast({html: myHTML, displayLength: 10000})">
             <i class="material-icons left">sentiment_dissatisfied</i>@lang('general.report_bug')</a></li>
+    @if(Auth::user())
+    <li>
+        <a href="{{ route('logout') }}"  onclick="event.preventDefault(); document.getElementById('logout-form').submit();">
+            <i class="material-icons left">login</i>@lang('general.logout')
```

Comment:
You could use exit_to_app icon for this (just a little better - it's nonsense that google doesn't have a logout icon :) )